src/HomeWork/HomeWork4/part1/studentTest.java

```diff
@@ -0,0 +1,17 @@
+package HomeWork.HomeWork4.part1;
+
+public class studentTest {
+    public static void main(String[] args) {
+        student st1 = new student(1, "Alan", "Wake", 3, 70, 60, 100);
+        student st2 = new student(2, "Rose", "Marigold", 5, 90, 70, 80);
+        student st3 = new student(3, "Barry", "Wheeler", 4, 100, 80, 80);
+
+        int gpa1 = (st1.gpaEconomy + st1.gpaEnglish + st1.gpaMath) / 3;
```

Comment:
Here we duplicate our actions so it's needed to create a method for average score calculation

Reply:
Please add a method like this:

 public double calculateAverageGrade() {
        return (mathGrade + economicsGrade + foreignLanguageGrade) / 3;
    }